app/models/user.rb

```diff
@@ -500,6 +500,11 @@ def reload(...)
     super(...)
   end
 
+  # User was flagged by TMX as fraudulent, AND failed at the Post Office.
+  def double_failed?
+      fraud_review_pending? && %w[cancelled failed].include?(in_person_enrollments&.last&.status)
+  end
```

Comment:
Method probably needs to be renamed, but this kind of odd set of checks makes sense on the User object IMHO.

Reply:
i kind of like the naming